go/vt/sqlparser/parse_test.go

```diff
@@ -1510,6 +1510,33 @@ func TestValid(t *testing.T) {
 	}
 }
 
+func TestCreateViewSelectPosition(t *testing.T) {
+	cases := []struct {
+		query string
+		sel   string
+	}{{
+		query: "create view a as select current_timestamp()",
+		sel:   "select current_timestamp()",
+	},{
+		query: "create view a as select /* comment */ 2 + 2 from dual",
+		sel:   "select /* comment */ 2 + 2 from dual",
+	}}
+	for _, tcase := range cases {
+		tree, err := Parse(tcase.query)
+		if err != nil {
+			t.Errorf("Parse(%q) err: %v", tcase.query, err)
+		}
+		ddl, ok := tree.(*DDL)
+		if !ok {
+			t.Errorf("Expected DDL when parsing (%q)", tcase.query)
+		}
+		sel := tcase.query[ddl.SelectPositionStart:ddl.SelectPositionEnd-1]
```

Comment:
Thoughts on this `ddl.SelectPositionEnd-1`? Should the rule actually be `SelectPositionEnd: $8 - 1`?